backend/swagger/v1/swagger.yaml

```diff
@@ -324,20 +331,20 @@ paths:
           required: true
       responses:
         '200':
-          description: Ready surveys
+          description: Successful operation - Survey with a given ID returned.
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Survey'
   '/api/v1/answers_surveys':
     post:
-      summary: Create Answers Survey
-      description: Create Answers Survey
+      summary: Add a new Answers Survey to the store
```

Comment:
This one I would return to `Create Answers Survey`.
Reasons
- The term `Add` usually is related to `array` or `list` data structures
- The term `store` usually is related to a storage pattern in front ent.